test/redox/models/medications_test.rb

```diff
@@ -28,8 +28,13 @@ class MedicationsTest < Minitest::Test
 
       it 'can be built' do
         m = Redox::Models::Medications.new
-        m.administrations = [{"Status"=>"Complete", "Medication"=>{"Product"=>{"Code"=>"12341234"}}}]
+        m.administrations = [{"Status"=>"Complete", "Medication"=>{"Product"=>{"Code"=>"12341234"}, "Components" => [], "OrderedBy" => {}, "Indications" => [], "AdministeringProvider" => {}}}]
+        med = m.administrations.first["Medication"]
         assert_equal('Complete', m.Administrations.first['Status'])
+        assert_equal([], med["Components"])
+        assert_equal({}, med["OrderedBy"])
+        assert_equal([], med["Indications"])
+        assert_equal({}, med["AdministeringProvider"])
```

Comment:
I like the direction this test is going with!  One thing that was noticed was that the updates to the model was to the singular `Medication` model but this test is testing against the plural `Medications` model.  It might be a good pathway to create a new `medication_test.rb` and using a `Redox::Models::Medication` record to test against